hedera-mirror-monitor test case ConfigurableTransactionGeneratorTest.receiptPercent failed

<!--  Thanks for submitting a bug report!  Before submitting:
1. Try searching the existing issues to see if your issue has already been reported
2. Try asking on our Discord group (https://discordapp.com/invite/FFb9YFX)
3. If you're reporting a security vulnerability, please email security@hedera.com instead of opening an issue
-->

**Detailed Description**
The test case `ConfigurableTransactionGeneratorTest.receiptPercent` failed in CircleCI:

```
[ERROR] com.hedera.mirror.monitor.generator.ConfigurableTransactionGeneratorTest.receiptPercent  Time elapsed: 5.373 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting:
  <0.0022>
to be close to:
  <0.001>
by less than <0.001> but difference was <0.0012000000000000001>.
(a difference of exactly <0.001> being considered valid)
	at com.hedera.mirror.monitor.generator.ConfigurableTransactionGeneratorTest.receiptPercent(ConfigurableTransactionGeneratorTest.java:130)
```

**Actual Behavior**

**Expected Behavior**
The test case passes

**Environment:**

**Additional Context**

--- a/hedera-mirror-monitor/src/test/java/com/hedera/mirror/monitor/generator/ConfigurableTransactionGeneratorTest.java
+++ b/hedera-mirror-monitor/src/test/java/com/hedera/mirror/monitor/generator/ConfigurableTransactionGeneratorTest.java
@@ -117,7 +117,7 @@ class ConfigurableTransactionGeneratorTest {
 
     @Test
     void receiptPercent() {
-        properties.setReceipt(0.001);
+        properties.setReceipt(0.1);
         Multiset<Boolean> receipts = HashMultiset.create();
 
         for (int i = 0; i < SAMPLE_SIZE; ++i) {
@@ -127,7 +127,7 @@ class ConfigurableTransactionGeneratorTest {
         assertThat((double) receipts.count(true) / SAMPLE_SIZE)
                 .isNotNegative()
                 .isNotZero()
-                .isCloseTo(properties.getReceipt(), within(0.001));
+                .isCloseTo(properties.getReceipt(), within(properties.getReceipt() * 0.2));
     }
 
     @Test
@@ -162,7 +162,7 @@ class ConfigurableTransactionGeneratorTest {
         assertThat((double) records.count(true) / SAMPLE_SIZE)
                 .isNotNegative()
                 .isNotZero()
-                .isCloseTo(properties.getRecord(), within(SAMPLE_SIZE * 0.05));
+                .isCloseTo(properties.getRecord(), within(properties.getRecord() * 0.2));
     }
 
     @Test

